<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-U64FBT2" failures="0" tests="1" name="general_settings_backend.VerifyActionsCity" time="4.933" errors="1" timestamp="2021-09-01T16:10:08 BDT" skipped="0">
  <testcase classname="general_settings_backend.VerifyActionsCity" name="ActionsTest" time="4.933">
    <error message="unknown error: Chrome failed to start: exited normally.
  (chrome not reachable)
  (The process started from chrome location C:\Program Files\Google\Chrome\Application\chrome.exe is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Build info: version: &#039;3.5.3&#039;, revision: &#039;a88d25fe6b&#039;, time: &#039;2017-08-29T12:54:15.039Z&#039;
System info: host: &#039;DESKTOP-U64FBT2&#039;, ip: &#039;192.168.88.26&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;16.0.2&#039;
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
	Ordinal0 [0x00E53733+2504499]
	Ordinal0 [0x00DEC401+2081793]
	Ordinal0 [0x00CF2628+1058344]
	Ordinal0 [0x00D0C99F+1165727]
	Ordinal0 [0x00D09DC3+1154499]
	Ordinal0 [0x00D39B82+1350530]
	Ordinal0 [0x00D397DA+1349594]
	Ordinal0 [0x00D35D4B+1334603]
	Ordinal0 [0x00D122B4+1188532]
	Ordinal0 [0x00D13149+1192265]
	GetHandleVerifier [0x00FCFB8C+1512252]
	GetHandleVerifier [0x0107B0DF+2214031]
	GetHandleVerifier [0x00ED4BC3+484211]
	GetHandleVerifier [0x00ED3E69+480793]
	Ordinal0 [0x00DF218D+2105741]
	Ordinal0 [0x00DF66E8+2123496]
	Ordinal0 [0x00DF6827+2123815]
	Ordinal0 [0x00DFFB73+2161523]
	BaseThreadInitThunk [0x7552FA29+25]
	RtlGetAppContainerNamedObjectPath [0x77537A7E+286]
	RtlGetAppContainerNamedObjectPath [0x77537A4E+238]
" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: unknown error: Chrome failed to start: exited normally.
  (chrome not reachable)
  (The process started from chrome location C:\Program Files\Google\Chrome\Application\chrome.exe is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Build info: version: '3.5.3', revision: 'a88d25fe6b', time: '2017-08-29T12:54:15.039Z'
System info: host: 'DESKTOP-U64FBT2', ip: '192.168.88.26', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '16.0.2'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00E53733+2504499]
Ordinal0 [0x00DEC401+2081793]
Ordinal0 [0x00CF2628+1058344]
Ordinal0 [0x00D0C99F+1165727]
Ordinal0 [0x00D09DC3+1154499]
Ordinal0 [0x00D39B82+1350530]
Ordinal0 [0x00D397DA+1349594]
Ordinal0 [0x00D35D4B+1334603]
Ordinal0 [0x00D122B4+1188532]
Ordinal0 [0x00D13149+1192265]
GetHandleVerifier [0x00FCFB8C+1512252]
GetHandleVerifier [0x0107B0DF+2214031]
GetHandleVerifier [0x00ED4BC3+484211]
GetHandleVerifier [0x00ED3E69+480793]
Ordinal0 [0x00DF218D+2105741]
Ordinal0 [0x00DF66E8+2123496]
Ordinal0 [0x00DF6827+2123815]
Ordinal0 [0x00DFFB73+2161523]
BaseThreadInitThunk [0x7552FA29+25]
RtlGetAppContainerNamedObjectPath [0x77537A7E+286]
RtlGetAppContainerNamedObjectPath [0x77537A4E+238]

at jdk.internal.reflect.GeneratedConstructorAccessor15.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$new$0(W3CHandshakeResponse.java:57)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$getResponseFunction$2(W3CHandshakeResponse.java:104)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$24(ProtocolHandshake.java:359)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:362)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:136)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:82)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:646)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:255)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:237)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:138)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:178)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:167)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:124)
at general_settings_backend.VerifyActionsCity.ActionsTest(VerifyActionsCity.java:13)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- ActionsTest -->
  <system-out/>
</testsuite> <!-- general_settings_backend.VerifyActionsCity -->
